Libs/vtkSegmentationCore/vtkCalculateOversamplingFactor.cxx

```diff
@@ -463,7 +463,7 @@ void vtkCalculateOversamplingFactor::ApplyOversamplingOnImageGeometry(vtkOriente
       {
       int dimension = extent[axis*2+1] - extent[axis*2] + 1;
       int extentMin = static_cast<int>(ceil(oversamplingFactor * extent[axis * 2]));
```

Comment:
I'm not sure as I did it long ago, but back then I tested everything with the tiny patient and checked voxel by voxel, so this should be OK. If you want to double check, feel free to do it, write a test, or add a ticket.

Reply:
I was just wondering if it may crop boundary voxels. I don't remember having seen any unwanted cropping, but also I did not use oversampling that much. Now that it'll be more easily available, I'll have more chance to use it and confirm that it works well.

Reply:
Sounds good, thanks!